database/migrations/20221102141110-create-user.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
+        type: Sequelize.STRING
+      },
+      lastName: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      avatar: {
+        type: Sequelize.STRING
+      },
+      roleId: {
+        type: Sequelize.INTEGER
```

Comment:
### Pending

add the foreign key in the migration:

` roleId: {
        type: Sequelize.INTEGER,
references: {
          model: {
            tableName: 'Role',
            schema: 'schema'
          },
          key: 'id'
        },
        allowNull: false
},`